### DISCUSSION TITLE
serial sensor input

### DISCUSSION THREAD

**Discussion by aphexcloud** [Rem0o/FanControl.Releases#1450] [Ideas]
is there a chance that fancontrol could read celcius temp from arduino or other MCU

**Comment by Rem0o** [comment]
Not directly as of now. See https://github.com/Rem0o/FanControl.Releases/discussions/1422#discussioncomment-4645878.

You could also make a very simple plugin that does the serial communication. It's a 30 minute project if you know what you are doing.

**Reply by aphexcloud** [reply]
sadly i do not, but its somthing i am looking into to see  if can fumble my way through thank you for the quick responce and amazing software

**Reply by secretformula** [reply]
https://github.com/secretformula/FanControl.UsbTemp
